docs/releases/5.0.txt

```diff
@@ -139,6 +139,11 @@ Minor features
 
 * ``XRegExp`` is upgraded from version 3.2.0 to 5.1.1.
 
+* ``AdminSite.site_header`` is now displayed with ``<div id="site-name">`` instead of
+  ``<h1 id="site-name">``. Screen readers use heading elements for navigation within
+  a page, and the actual site header isn't relevant to the page's content as it is
+  repeated on all pages.
```

Comment:
```suggestion
* ``AdminSite.site_header`` is now displayed with ``<div id="site-name">`` instead of
  ``<h1 id="site-name">``. Screen reader users rely on heading elements for navigation within
  a page, and the site header isn't helpful as it is repeated on all pages.
```

Just a few suggested improvements for clarity